extension.Rmd

```diff
@@ -0,0 +1,175 @@
+data <- rename(data, max_er = rmax) # Maximum toughness in extreme right discourse
+
+```
+
+
+# Preparing data for analysis
+
+```{r}
+# Create centered variables for Saliance and Variance based on grand mean centering according to author
+
+data <- data  %>% 
+  mutate(salience_mean_c = (salienzmean - 3.84568)) %>% 
+  mutate(var_c = rvar - 21.75423)
+
+# Make male variable numeric
+data <- data %>%
+  mutate(male = as.numeric(male))
+
+# Create a dummy variable for country (transform from character to factor variable)
+data <- data %>%
+  mutate(country = factor(sortcountry, labels = c("AT", "BE","DE-E","DE-W","DK","ES","FI","FR","GR","IT","LU","NL","NO","PT","SE")))
+```
+
+
+# Model 1: Adorno Thesis (Personality Traits and Value Orientations)
+The first model is based on Theodor Adorno's 1950 theory that support for extreme right parties (in his case, fascist parties) originates from supporters' personality traits and value orientations, which make individuals more receptive to the appeals of extreme right parties.  We operationalize "personality traits and value orientations" using the individual level characteristics from our data set: gender, age, occupation, marital status, and self-placement on a left-right scale. 
+
+```{r}
+library(separationplot)
+library(Hmisc) 
+
+model_1 <- glm(ext_vote ~ male + age_1 + age_2 + age_4 + edu_1 + edu_2 + farmer_own + worker + retired + unemployed + partner + left_right_scale + country - 1, data = data)
+summary(model_1)
+actual <- model_1$y # Create vector with actual values for vote intention for extreme right parties
+
+log.odds <- model_1$fitted.values # Create a vector with log-odd values
+# Create link function to transform log-odds into predicted probabilities
+link <- function(P) {
+  1/(1+exp(-P))
+} 
+pred <- link(log.odds) # Transform log-odds into predicted probabilities
+pred <- as.vector(pred) 
+range(pred) # variation in the predicted probabilities: very little variation.
+dev.off()
+# Create separation plot to visualize the predictive power of the model
+separationplot(pred = pred, actual = actual, type = "rect", heading = "Model1: The Adorno Thesis", xlab = "ER Vote Intention", newplot = F, line = T, lwd2 = 0.8, height = 0.5, show.expected = T) 
+```
+
+![Model 1](figures/SP.Model1.png)
+
+#Conclusion
+There isn't a big range in the predicted probabilities of the model, therefore the predicted probabilities are not very helpful. Separation plot helps a lot to discern the predictive power of the model. As the events of extreme right vote intension increases as we move to the right (i.e. red lines cluster around the end of the graph), the model does a good job of estimating the likelihood of events. However, as the small arrow in the graph shows, the model predicts that half the number of observations will have the intention to vote for an extreme right party. The number of actual events (red lines) is much lower.  
+
```

Comment:
I think that the range of the predictions and the order of the predictions are different.  In other words, I think you have two findings here not just one.